elasticdl/python/master/embedding_service.py

```diff
@@ -259,9 +259,7 @@ def start_embedding_pod_and_redis(
         # and should not fix ports
         address_ip = pod.status.pod_ip
         while not address_ip:
-            pod = self._k8s_client.get_embedding_service_pod(
-                embedding_service_id
-            )
+            pod = k8s_client.get_embedding_service_pod(embedding_service_id)
```

Comment:
should we wait for a while before the next check?

Reply:
That's not relevant to this PR. Perhaps consider creating a new issue to discuss?

Reply:
:smiley: